src/main/java/com/exasol/bucketfs/BucketConstants.java

```diff
@@ -4,8 +4,17 @@
  * BucketFS constants.
  */
 public final class BucketConstants {
+    /**
+     * path separator
+     */
```

Comment:
@redcatbear @kaklakariada  do we have a style for these?
I usually do: ` /** Path separator */`
So one line, starting with upper, no-dot. But I don't know if thats just me ;-)

Reply:
The Eclipse formatter does not enforce the single-line-style, but when you type it as a single line, the formatter won't change it. So you will have to manually re-format these comments.

Reply:
I do to. I am assuming a formatter did this.

Reply:
@redcatbear let me precise the quiestion. Is it ok like it is here (muti-line, starting lower), or do we need to change it?

Reply:
This is what the default 'add javadoc' action in intellij does

Reply:
changed them to single line with uppercase